[2022-09-02 00:09:55,846] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.final_dag scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-09-02 00:09:55,868] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.final_dag scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-09-02 00:09:55,869] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-09-02 00:09:55,869] {taskinstance.py:1377} INFO - Starting attempt 2 of 2
[2022-09-02 00:09:55,870] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-09-02 00:09:55,892] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): final_dag> on 2022-09-01 00:00:00+00:00
[2022-09-02 00:09:55,901] {standard_task_runner.py:52} INFO - Started process 28634 to run task
[2022-09-02 00:09:55,905] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'final_dag', 'scheduled__2022-09-01T00:00:00+00:00', '--job-id', '307', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmpcud4lqoo', '--error-file', '/tmp/tmpvt30xg_k']
[2022-09-02 00:09:55,906] {standard_task_runner.py:80} INFO - Job 307: Subtask final_dag
[2022-09-02 00:09:55,998] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.final_dag scheduled__2022-09-01T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-09-02 00:09:56,109] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=final_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-09-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-01T00:00:00+00:00
[2022-09-02 00:09:56,189] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/scripts/output_df.py", line 20, in final_df
    new_df = df.query('Year>= 1960 & Year <1980 & Make == ["Ford", "Dodge", "GMC", "Mercury", "Lincoln", "Chrysler", "Chevrolet"]')
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/frame.py", line 4060, in query
    res = self.eval(expr, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/frame.py", line 4191, in eval
    return _eval(expr, inplace=inplace, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/eval.py", line 348, in eval
    parsed_expr = Expr(expr, engine=engine, parser=parser, env=env)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 806, in __init__
    self.terms = self.parse()
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 825, in parse
    return self._visitor.visit(self.expr)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 411, in visit
    return visitor(node, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 417, in visit_Module
    return self.visit(expr, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 411, in visit
    return visitor(node, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 420, in visit_Expr
    return self.visit(node.value, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 411, in visit
    return visitor(node, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 745, in visit_BoolOp
    return reduce(visitor, operands)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 738, in visitor
    lhs = self._try_visit_binop(x)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 734, in _try_visit_binop
    return self.visit(bop)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 411, in visit
    return visitor(node, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 718, in visit_Compare
    return self.visit(binop)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 411, in visit
    return visitor(node, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 534, in visit_BinOp
    return self._maybe_evaluate_binop(op, op_class, left, right)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 528, in _maybe_evaluate_binop
    return self._maybe_eval(res, eval_in_python + maybe_eval_in_python)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 489, in _maybe_eval
    self.env, self.engine, self.parser, self.term_type, eval_in_python
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/ops.py", line 433, in evaluate
    res = self(env)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/computation/ops.py", line 413, in __call__
    return self.func(left, right)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/arraylike.py", line 52, in __ge__
    return self._cmp_method(other, operator.ge)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/series.py", line 5502, in _cmp_method
    res_values = ops.comparison_op(lvalues, rvalues, op)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 284, in comparison_op
    res_values = comp_method_OBJECT_ARRAY(op, lvalues, rvalues)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 73, in comp_method_OBJECT_ARRAY
    result = libops.scalar_compare(x.ravel(), y, op)
  File "pandas/_libs/ops.pyx", line 107, in pandas._libs.ops.scalar_compare
TypeError: '>=' not supported between instances of 'str' and 'int'
[2022-09-02 00:09:56,210] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=csv_dag, task_id=final_dag, execution_date=20220901T000000, start_date=20220902T000955, end_date=20220902T000956
[2022-09-02 00:09:56,224] {standard_task_runner.py:97} ERROR - Failed to execute job 307 for task final_dag ('>=' not supported between instances of 'str' and 'int'; 28634)
[2022-09-02 00:09:56,238] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-09-02 00:09:56,275] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
